test/test_torch.py

```diff
@@ -1288,6 +1288,14 @@ def test_tensor_factory(self):
         self.assertEqual(res1, expected)
         self.assertIs(torch.int, res1.dtype)
 
+        # test copy with numpy
+        if TEST_NUMPY:
+            a = np.array([5.])
```

Comment:
Does this return a `float64` array? If that's the case, the old code-path would also return a copy of the array, right?
Maybe it's better to impose that this is a `float32` `np.array`?

Reply:
Nope, this is right, because we set the default tensor type to be `double` in `common.py`. Sorry for the noise